### ISSUE TITLE
problems with Beta 0.106.0b0

### ISSUE THREAD

**Issue by VDRainer** [gadgetchnnel/lovelace-home-feed-card#29]
![Auswahl_270](https://user-images.githubusercontent.com/26381449/74862947-4e54f800-534d-11ea-9a11-0ea444367d2d.png)
I think related to:
https://developers.home-assistant.io/blog/2020/02/18/106-custom-card-changes.html

**Comment by gadgetchnnel** [comment]
I've just deployed a new beta (0.3.9b1) which should hopefully fix this issue (I'm still on 0.105 myself, so I can't actually test it myself) can you give it a go?

**Comment by VDRainer** [comment]
@gadgetchnnel , sorry, no change.
![Auswahl_271](https://user-images.githubusercontent.com/26381449/74866690-8e1ede00-5353-11ea-83b3-99942069b99a.png)

**Comment by gadgetchnnel** [comment]
@VDRainer Can you click on the "lovelace-home-feed-card.js:365" link in the error and let me know what you see on that line when the Javascript opens in the developer tools?

You should see:

`entityConf = { ...entityConf, exclude_states: ["unknown"]};`

if you see:

`entityConf.exclude_states = ["unknown"];`

you probably have an the old Javascript cached (or HACS didn't download the new version properly) so you should probably try a hard refresh and/or reinstalling it via HACS.

**Comment by VDRainer** [comment]
Yeah! After reinstalling the red card is gone, but card shows nothing.
Should it? :smile: 
![Auswahl_272](https://user-images.githubusercontent.com/26381449/74868546-b825cf80-5356-11ea-9786-38963917c3f9.png)
![Auswahl_273](https://user-images.githubusercontent.com/26381449/74868661-e1def680-5356-11ea-9856-fbd71bd4af8a.png)
Thanks for your effort!

**Comment by gadgetchnnel** [comment]
> Yeah! After reinstalling the red card is gone, but card shows nothing.
> Should it? 😄

Those errors look like it is failing to import two modules that it needs. It looks like these are somehow being sent with a MIME type of "application/octet-stream" rather than "text/javascript". I wonder if this is an issue with how HACS is sending these.

I'm just updating the 0.106.0b0 myself, so I'll know soon if that is also 0.106 related.

**Comment by VDRainer** [comment]
@gadgetchnnel , you're right, installed the files manually and eveything works!
![Auswahl_274](https://user-images.githubusercontent.com/26381449/74870309-d0e3b480-5359-11ea-8b2f-7f8eeb757e10.png)

**Comment by VDRainer** [comment]
Maybe one more info.
Had the same issue (card shows no entities) in my prod system (0.105.5 reverted back from 0.106 beta) after installing the 0.3.9b1 and reverting back to 0.3.8.
Nothing helped, had to install the files manually and all workes again.

**Comment by Mediacj** [comment]
I have the same problem after installing the beta, but after I placed the two missing modules in the home-feed directory by hand it still doesn't work the screen stays empty and no errors?

EDIT: solved now I replaced all the files in the home-feed dir and it works again! Thx for this great script!
EDIT2: it shows up but doesn't get updated it stays static? To be clear I'am on 0.3.8 now.
EDIT3: a restart of Home Assistant made it work again!
